conda_smithy/cli.py

```diff
@@ -248,6 +250,55 @@ def __call__(self, args):
         )
 
 
+class AddAzureBuildId(Subcommand):
+    subcommand = "azure-buildid"
+
+    def __init__(self, parser):
+        # conda-smithy azure-buildid ./
+        super(AddAzureBuildId, self).__init__(
+            parser,
+            dedent("""\
+                Update the azure configuration stored in the config file.
+                This command is affected by the following environment variables:
+                    AZURE_ORG_OR_USER
+                    AZURE_PROJECT_NAME
+                """
+            )
+        )
+        scp = self.subcommand_parser
+        scp.add_argument(
+            "--feedstock_directory",
+            default=os.getcwd(),
+            help="The directory of the feedstock git repository.",
+        )
+        group = scp.add_mutually_exclusive_group()
+        group.add_argument(
+            "--user", help="github username under which to register this repo"
```

Comment:
```suggestion
            "--user", help="azure username under which to register this repo"
```

Reply:
The github user is not the same as the azure user.

Reply:
Sure, but isn't that what this option is used for? For the azure user?

Reply:
its actually just ignored.  should i remove it for simplicity?